lfs/config.go

```diff
@@ -409,16 +415,26 @@ func (c *Configuration) loadGitConfig() bool {
 	c.gitConfig = make(map[string]string)
 	c.extensions = make(map[string]Extension)
 	uniqRemotes := make(map[string]bool)
+	uniqInsteadOfs := make(map[string]string)
+	c.insteadOfs = make(map[string]string)
 
 	configFiles := []string{
 		filepath.Join(LocalWorkingDir, ".lfsconfig"),
 
 		// TODO: remove .gitconfig support for Git LFS v2.0 https://github.com/github/git-lfs/issues/839
 		filepath.Join(LocalWorkingDir, ".gitconfig"),
 	}
+
+	listOutput, err := git.Config.ListRegexp(`^url.*`)
+	if err != nil {
+		panic(fmt.Errorf("Error listing git config with regexp `^url.*`: %s", err))
+	}
+	c.readGitInsteadOfConfig(listOutput, uniqInsteadOfs)
+	c.insteadOfs = uniqInsteadOfs
```

Comment:
Excuse my thinko there. Fixed. We don't really need the `--get-regexp` function anymore; do you want to keep it around anyway?